gwpy/cli/cliproduct.py

```diff
@@ -639,13 +640,14 @@ def set_legend(self):
         return leg
 
     def set_title(self, title):
-        """Set the title for this plot.
+        """Set the title(s) for this plot.
 
         The `Axes.title` actually serves at the sub-title for the plot,
         typically giving processing parameters and information.
         """
+
         if title is None:
-            title = self.get_title().rstrip(', ')
+            title = [self.get_title().rstrip(', ')]
```

Comment:
Because we now don't parse `--title` as a `list`, this needs to be just:

```suggestion
            title = self.get_title().rstrip(', ')
```

Reply:
done